docs/Contributors_Guide/add_use_case.rst

```diff
@@ -209,56 +209,425 @@ Add Sphinx Documentation File
+    # GenEnsProd are 29 files containing the uncalibrated probability forecasts for 
+    # the lower tercile of January for each year. The final probability verification 
+    # is done across the temporal scale in SeriesAnalysis, and the spatial scale in GridStat.
+
+
+    METplus Configuration
+
+    This section has set language that describes how all of the configuration files 
+    are loaded into METplus, followed by what’s passed in by the user at runtime, 
+    and used to produce the final configuration file that controls the METplus run. 
+    It concludes with an embedded link (and image) of the user’s configuration file 
+    the use case will run. The only two pieces that will change are the path to the 
+    use case’s configuration file, and the embedded configuration file for the use case.
+
+    Example:
+
+    ##############################################################################
+    # METplus Configuration
+    # ---------------------
+    #
+    # METplus first loads all of the configuration files found in parm/metplus_config, 
+    # then it loads any configuration files passed to METplus via the command line, 
+    # i.e. parm/use_cases/model_applications/s2s/SeriesAnalysis_fcstCFSv2_obsGHCNCAMS_climoStandardized_MultiStatisticTool.conf
+    #
+    # .. highlight:: bash
+    # .. literalinclude:: ../../../../parm/use_cases/model_applications/s2s/SeriesAnalysis_fcstCFSv2_obsGHCNCAMS_climoStandardized_MultiStatisticTool.conf
+    #
+
+
+    MET Configuration
+    
+    This section has set language that describes how settings in the MET configuration file 
+    will ultimately be used to run METplus, along with any changes made from the default 
+    by the user’s configuration file. It concludes with an embedded link (and image) of 
+    the default configuration file(s) for all MET tool(s) used. The only change that needs 
+    to occur for this section is which MET configuration file is embedded.
+    If no MET tool(s) are used, that should be noted here and replace the default language.
+
+    Example:
+
+    ##############################################################################
+    # MET Configuration
+    # -----------------
+    #
+    # METplus sets environment variables based on user settings in the METplus 
+    # configuration file. See :ref:`How METplus controls MET config file settings<metplus-control-met>` for more details.
+    #
+    # **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!**
+    #
+    # If there is a setting in the MET configuration file that is currently not supported by METplus you’d like to control, please refer to:
+    # :ref:`Overriding Unsupported MET config file settings<met-config-overrides>`
+    #
+    # .. highlight:: bash
+    # .. literalinclude:: ../../../../parm/met_config/GridStatConfig_wrapped
+
+
+    Python Embedding
```

Comment:
Please break up this section so the user doesn't have to horizontally scroll to read it